exercises/protein-translation/package.yaml

```diff
@@ -0,0 +1,20 @@
+name: protein-translation
+version: 1.1.0.1
```

Comment:
1.0.1.1 because https://github.com/exercism/problem-specifications/blob/master/exercises/protein-translation/canonical-data.json#L3 says 1.0.1 (unless it changes after I wrote this comment)